### DISCUSSION TITLE
脉冲式工作 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#434] [General]
# 脉冲式工作 - Miao Yu | 于淼

https://yufree.cn/cn/2023/09/23/implus-working/

**Comment by Jpzhaoo** [comment]
我也是脑力劳动多了就喜欢做家务。大概就是脑子累了，就得身体活动起来，平衡一下。

**Reply by yufree** [reply]
体力劳动产生的成就感似乎比无聊的脑力劳动更多，毕竟体力劳动上的预期是可以完成的，脑力任务很多不一定有结果。

**Comment by KexingWestRoad** [comment]
体力劳动更多的是身体累，但是会让自己很快乐。
脑力劳动更多的是心累，脑力成功之后去总结的时候，总感觉不确定性很强，是随机的运气，没有什么经验之谈。

**Comment by KexingWestRoad** [comment]
《现代科研指北》已加入书架。
